applications/commerce-and-marketplaces.md

```diff
@@ -0,0 +1,19 @@
+---
+description: Machine translation applied to the e-commerce field
+---
+
```

Comment:
# Commerce and marketplaces

Machine translation for **for commerce and marketplaces** is translation of product content, like product titles and descriptions, into the languages of consumers.

The goals are:
- sales conversion
- [SEO](seo) and site search
- customer support

The top merchants and platforms cannot human-translate all products into all languages, because of the scale.

The translation of product titles and descriptions is handled by the platforms, merchants or brands themselves.

## Content types

- Product titles
- Product descriptions
- Product reviews
- Customer support [chat](live-chat) and email